src/CMake/FindADIOS2.cmake

```diff
@@ -29,7 +29,12 @@ ELSE()
     SET(LIB lib)
 ENDIF()
 
-SET_UP_THIRD_PARTY(ADIOS2 ${LIB} include adios2 adios2_atl adios2_dill adios2_evpath adios2_ffs adios2_sst taustubs)
+if(NOT WIN32)
```

Comment:
On kickit I didn't have the rdma libraries installed, which forced me to turn off SST support and then I only got the 2 libraries you list here. I was able to find the rdma source code on GitHub and I built it and then linked against that. I haven't checked any of those changes in and I'll hold off until I talk to Dave about this, but I thought I'd mention it as a possible route to enabling SST on Windows, if the issue was the rdma libraries. This is just a comment and you don't need to do anything about this.